scripts/install_third_party_libs_test.py

```
@@ -259,6 +265,22 @@ def mock_isdir(path: str) -> bool:
+        self.assertIn(yarn_install_command, self.all_cmd_tokens)
+        self.assertNotIn('--ignore-engines', yarn_install_command)
+        yarn_install_index = self.all_cmd_tokens.index(yarn_install_command)
+        yarn_install_env = self.all_envs[yarn_install_index]
+        self.assertIsNotNone(yarn_install_env)
+        assert yarn_install_env is not None
+        expected_node_20_bin_path = os.path.join(
+            common.PLAYWRIGHT_NODE_PATH, 'bin'
+        )
+        self.assertTrue(
+            yarn_install_env['PATH'].startswith(
+                '%s%s' % (expected_node_20_bin_path, os.pathsep)
```

Ruff (0.16.2)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)